quickstart/101-aci-linuxcontainer-public-ip/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=0.12"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>2.0"
```

Comment:
Is there a reason that this is 2.0? If not, it should be updated to a more current version.

Reply:
Changed.